<launch>
  	<arg name="div" default="false" />
  	<arg name="normal" default="false" />  
	<group if="$(arg div)">
		<node pkg="measurment_controller" type="measurment_client_node_div" name="measurment_client_node_div" output="screen">
			<rosparam file="$(find measurment_controller)/config/config.yaml" command="load" />
		</node>
	</group>
	<group if="$(arg normal)">
		<node pkg="measurment_controller" type="measurment_client_node_normal" name="measurment_client_node_normal" output="screen">
			<rosparam file="$(find measurment_controller)/config/config.yaml" command="load" />
		</node>
	</group>
 </launch>

